assets/js/googlesitekit/data/utils.js

```diff
@@ -206,6 +212,51 @@ export const collectName = ( ...args ) => {
 	return names.shift();
 };
 
+/**
+ * An object of common actions most stores will use.
+ *
+ * @since n.e.x.t
+ *
+ * @return {Object} key/value list of common actions most stores will want.
+ */
+export const commonActions = {
+	/**
+	 * Dispatches an action and calls a control to get the current data registry.
+	 *
+	 * Useful for controls and resolvers that wish to dispatch actions/use selectors
+	 * on the current data registry.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {Object} FSA-compatible action.
+	 */
+	getRegistry() {
+		return { type: GET_REGISTRY };
+	},
+	initialize: initializeAction,
+};
+
+/**
+ * An object of common controls most stores will use.
+ *
+ * @since n.e.x.t
+ *
+ * @return {Object} key/value list of common controls most stores will want.
+ */
+export const commonControls = {
+	/**
+	 * Returns the current registry.
+	 *
+	 * Useful for controls and resolvers that wish to dispatch actions/use selectors
+	 * on the current data registry.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {Object} FSA-compatible action.
+	 */
+	[ GET_REGISTRY ]: createRegistryControl( ( registry ) => () => registry ),
```

Comment:
Where does the registry parameter here come from?

Reply:
It's provided during store creation by `registry.registerStore > createNamespace > (createReduxStore)`
https://github.com/WordPress/gutenberg/blob/4857ad58c1241b3d63d21a6880c989b85746c3dc/packages/data/src/namespace-store/index.js#L129-L134